zerver/lib/integrations.py

```diff
@@ -402,9 +401,7 @@ def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
     WebhookIntegration("flock", ["customer-support"]),
     WebhookIntegration("freshdesk", ["customer-support"]),
     WebhookIntegration("freshping", ["monitoring"]),
-    WebhookIntegration(
-        "freshstatus", ["monitoring", "customer-support"]
-    ),
+    WebhookIntegration("freshstatus", ["monitoring", "customer-support"]),
```

Comment:
This junk was supposed to be part of the previous commit.

I've enjoyed setting `ruff-format` up to run in my editor on-save to avoid ever generating commits that don't pass that part of the linter.